index.js

```diff
@@ -1,204 +1,236 @@
-    if (!this.initialized) throw new Error(uninitializedError('removePushDeviceToken'))
+    if (!this.initialized)
+      throw new Error(uninitializedError("removePushDeviceToken"));
 
-    return RNMixpanel.removePushDeviceToken(deviceToken, this.apiToken)
+    return RNMixpanel.removePushDeviceToken(deviceToken, this.apiToken);
   }
 
   removeAllPushDeviceTokens(): Promise<void> {
-    if (!this.initialized) throw new Error(uninitializedError('removeAllPushDeviceTokens'))
+    if (!this.initialized)
+      throw new Error(uninitializedError("removeAllPushDeviceTokens"));
 
-    return RNMixpanel.removeAllPushDeviceTokens(this.apiToken)
+    return RNMixpanel.removeAllPushDeviceTokens(this.apiToken);
   }
 
   addPushDeviceToken(token: string): Promise<void> {
-    if (!this.initialized) throw new Error(uninitializedError('addPushDeviceToken'))
+    if (!this.initialized)
+      throw new Error(uninitializedError("addPushDeviceToken"));
 
-    return RNMixpanel.addPushDeviceToken(token, this.apiToken)
+    return RNMixpanel.addPushDeviceToken(token, this.apiToken);
   }
 
   // android only
   setPushRegistrationId(token: string): Promise<void> {
-    if (!this.initialized) throw new Error(uninitializedError('setPushRegistrationId'))
-
-    if (!RNMixpanel.setPushRegistrationId) throw new Error('No native implementation for setPushRegistrationId.  This is Android only.')
-    return RNMixpanel.setPushRegistrationId(token, this.apiToken)
+    if (!this.initialized)
+      throw new Error(uninitializedError("setPushRegistrationId"));
+
+    if (!RNMixpanel.setPushRegistrationId)
+      throw new Error(
+        "No native implementation for setPushRegistrationId.  This is Android only."
+      );
+    return RNMixpanel.setPushRegistrationId(token, this.apiToken);
   }
 
   // android only
   clearPushRegistrationId(): Promise<void> {
-    if (!this.initialized) throw new Error(uninitializedError('clearPushRegistrationId'))
-
-    if (!RNMixpanel.clearPushRegistrationId) throw new Error('No native implementation for setPusclearPushRegistrationIdhRegistrationId.  This is Android only.')
-    return RNMixpanel.clearPushRegistrationId()
+    if (!this.initialized)
+      throw new Error(uninitializedError("clearPushRegistrationId"));
+
+    if (!RNMixpanel.clearPushRegistrationId)
+      throw new Error(
+        "No native implementation for setPusclearPushRegistrationIdhRegistrationId.  This is Android only."
+      );
+    return RNMixpanel.clearPushRegistrationId();
   }
 
   reset(): Promise<void> {
-    if (!this.initialized) throw new Error(uninitializedError('reset'))
+    if (!this.initialized) throw new Error(uninitializedError("reset"));
 
-    return RNMixpanel.reset(this.apiToken)
+    return RNMixpanel.reset(this.apiToken);
   }
 
   createBooleanTweak(name: string, defaultValue: boolean): Promise<void> {
-    if (!this.initialized) throw new Error(NO_INSTANCE_ERROR)
+    if (!this.initialized) throw new Error(NO_INSTANCE_ERROR);
```

Comment:
`new Error(uninitializedError('createBooleanTweak'))` should be thrown instead if we stick to the convention above